test/acceptance/features/examples.feature

```diff
@@ -53,3 +54,54 @@ Feature: Verify examples provided in Service Binding Operator github repository
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "sbr-to-bind-hello-app-to-route" should be changed to "True"
         And jq ".status.conditions[] | select(.type=="Ready").status" of Service Binding "sbr-to-bind-hello-app-to-route" should be changed to "True"
         And Secret "sbr-to-bind-hello-app-to-route" contains "ROUTE_HOST" key with value "example-sbo.apps.ci-ln-smyggvb-d5d6b.origin-ci-int-aws.dev.rhcloud.com"
+
+    # https://github.com/redhat-developer/service-binding-operator/tree/master/examples/multiple_services
+    Scenario: Multiple Services Binding Example
+        Given DB "db-mul-svc-ex" is running
+        * Etcd operator running
+        * Etcd cluster "etcd-mul-svc-ex" is running
+        * "node-todo-mul-svc-ex" application is deployed from remote repository "https://github.com/akashshinde/node-todo.git"
+        When Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: sbr-bind-mul-svc-ex
+            spec:
+                application:
+                    name: node-todo-mul-svc-ex
+                    group: apps
+                    version: v1
+                    resource: deployments
+                services:
+                - group: postgresql.baiju.dev
+                  version: v1alpha1
+                  kind: Database
+                  name: db-mul-svc-ex
+                - group: etcd.database.coreos.com
+                  version: v1beta2
+                  kind: EtcdCluster
+                  name: etcd-mul-svc-ex
+                detectBindingResources: true
+            """
+        Then Service Binding "sbr-bind-mul-svc-ex" is ready
```

Comment:
Since we have mentioned application we can also add 
`And application should be re-deployed`